zipkin-gems/zipkin-tracer/lib/zipkin-tracer/version.rb

```diff
@@ -1,4 +1,4 @@
-# Copyright 2012 Twitter Inc.
+# Copyright 2015 Twitter Inc.
```

Comment:
nit: I don't think your change should impact copyrights. Typically projects only affect these when adding new files.